### PR TITLE
Adding ETHMilan 2024

### PR DESCRIPTION
**Pull request by gguideth**
Adding ETHMilan 2024 to Community Events List

### CODE REVIEW

File: src/data/community-events.json [hunk 1]
```diff
@@ -408,6 +408,16 @@
...
     "endDate": "2024-9-22",
     "imageUrl": "https://ethglobal.com/og.png"
+  },
+   {
+    "title": "ETHMilan",
+    "to": "https://ethmilan.xyz/",
+    "location": "Milan, Italy",
+    "description": "ETMilan returns for its second edition – join us for two days of networking, Ethereum talks & workshops, and an exciting Creative Hackathon!",
+    "startDate": "2023-10-05",
+    "endDate": "2023-10-06"
```
Comment (nhsz) [inline src/data/community-events.json:418]:
```suggestion
    "endDate": "2023-10-06",
```

File: src/data/community-events.json [hunk 2]
```diff
@@ -408,6 +408,16 @@
     "startDate": "2024-9-20",
     "endDate": "2024-9-22",
     "imageUrl": "https://ethglobal.com/og.png"
+  },
+   {
+    "title": "ETHMilan",
+    "to": "https://ethmilan.xyz/",
+    "location": "Milan, Italy",
+    "description": "ETMilan returns for its second edition – join us for two days of networking, Ethereum talks & workshops, and an exciting Creative Hackathon!",
+    "startDate": "2023-10-05",
```
Comment (nhsz) [inline src/data/community-events.json:417]: @gguideth Hi, `startDate` and `endDate` values seem to be wrong, thanks

**Review by nhsz — CHANGES_REQUESTED** [review]

File: src/data/community-events.json (hunk 2 above)
Comment (minimalsm) [inline src/data/community-events.json:417]:
```suggestion
    "startDate": "2024-9-26",
```

File: src/data/community-events.json [hunk 3]
```diff
@@ -408,6 +408,16 @@
...
+    "endDate": "2023-10-06",
```
Comment (minimalsm) [inline src/data/community-events.json:418]:
```suggestion
    "endDate": "2024-9-27",
```

File: src/data/community-events.json [hunk 4]
```diff
@@ -408,6 +408,16 @@
...
+    "imageUrl": "https://og.ethglobal.com/api/generic?text=Pragma%20San%20Francisco"
```
Comment (minimalsm) [inline src/data/community-events.json:419]:
```suggestion
    "imageUrl": "https://assets-global.website-files.com/65b77a0b9370aa6cf9f28462/65d329fd71a77f03daa10342_RectangleBannerWebsite_ETHMilan24_def.jpg"
```

File: src/data/community-events.json (hunk 2 above)
Comment (gguideth) [reply src/data/community-events.json]: snap, they are – let me fix this, sorry!

**Review by minimalsm — APPROVED** [review]
LGTM, thanks @gguideth

**Pull request merged**